public/app/plugins/panel/graph/module.ts

```diff
@@ -58,11 +57,8 @@ class GraphCtrl extends MetricsPanelCtrl {
     xaxis: {
       show: true,
       mode: 'time',
-      seriesValue: 'avg'
-    },
-    alert: {
-      warn: {op: '>', value: undefined},
-      crit: {op: '>', value: undefined},
+      name: null,
+      values: [],
```

Comment:
Why do you use array of value instead single value? Support of multiple value selection?